docs/SPRINTS.md

```diff
@@ -1,268 +1,342 @@
+_Deprecated._
```

Comment:
@ryanhofdotgov @michaelccata Should we just delete this file?

Reply:
+1 to delete. We have it elsewhere in Google Drive (though not as concisely).